### PR TITLE
added settings page, account selection menu styling,

### PR DESCRIPTION
**Pull request by fabiofederici**

### CODE REVIEW

**Review by 0x0ece — APPROVED** [review]

File: playground/src/app/products/[product]/page.tsx [hunk 1]
```diff
@@ -35,151 +32,76 @@ import SparkleColorMatcher, {
...
-  },
-  mgmtFee: {
-    label: "Management Fee",
-    color: "hsl(var(--chart-1))",
-  },
-} satisfies ChartConfig;
+const API_CONFIG = {
+  url: 'https://rpc.helius.xyz/', // Correct Helius RPC endpoint
+  apiKey: process.env.NEXT_PUBLIC_HELIUS_API_KEY || '', // Use environment variable
+};
```
Comment (yurushao) [inline playground/src/app/products/[product]/page.tsx:44]: We currently use 1 env variable for the RPC endpoint `NEXT_PUBLIC_SOLANA_RPC`

File: playground/src/app/products/[product]/page.tsx [hunk 2]
```diff
@@ -231,42 +149,364 @@ export default function ProductPage() {
...
+      const response = await fetch(`${API_CONFIG.url}?api-key=${API_CONFIG.apiKey}`, {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({
+          jsonrpc: '2.0',
+          id: '1',
+          method: 'getTokenAccounts',
+          params: {
```
Comment (yurushao) [inline playground/src/app/products/[product]/page.tsx:291]: this should work if we don't have too many holders of a share class.

When a mint has lots of holders, the results of the rpc call is paginated, see `limit` and `page` params:
https://docs.helius.dev/compression-and-das-api/digital-asset-standard-das-api/get-token-accounts

Comment (fabiofederici) [reply playground/src/app/products/[product]/page.tsx:291]: yes it goes all the way to 1000, figured that gives us enough time to find a proper solution

File: playground/src/app/products/[product]/page.tsx (hunk 1 above)
Comment (fabiofederici) [reply playground/src/app/products/[product]/page.tsx:44]: moved it directly to the fetch, not sure what you mean with the second point. this must be a helius key

**Pull request merged**